x/distribution/types/proposal.go

```diff
@@ -0,0 +1,74 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+const (
+	// ProposalTypeCommunityPoolSpend defines the type for a CommunityPoolSpendProposal
+	ProposalTypeCommunityPoolSpend = "CommunityPoolSpend"
+)
+
+// Assert CommunityPoolSpendProposal implements govtypes.Content at compile-time
+var _ govtypes.Content = CommunityPoolSpendProposal{}
+
+func init() {
+	govtypes.RegisterProposalType(ProposalTypeCommunityPoolSpend)
+	govtypes.RegisterProposalTypeCodec(CommunityPoolSpendProposal{}, "cosmos-sdk/CommunityPoolSpendProposal")
+}
+
+// CommunityPoolSpendProposal spends from the community pool
+type CommunityPoolSpendProposal struct {
+	Title       string         `json:"title"`
+	Description string         `json:"description"`
+	Recipient   sdk.AccAddress `json:"recipient"`
+	Amount      sdk.Coins      `json:"amount"`
```

Comment:
Here it is called an `amount`, but in `CommunityPoolSpendProposalReq` and `CommunityPoolSpendProposalJSON ` it is called a `deposit`?

Reply:
No, they are separate fields - `Deposit` is the usual deposit on a proposal, `Amount` is the amount spent from the community pool in a `CommunityPoolSpendProposal`.